index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://kit.fontawesome.com/620d418c84.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond&display=swap"
+      rel="stylesheet"
+    />
+    <title>Stock Exchange</title>
+  </head>
+  <body>
+    <div class="marquee visual-aid">
+      <span id="marqueeData"></span>
+    </div>
+
+    <h3 class="d-flex justify-content-center mt-5 text-light">
+      Search Nasdaq Stock
+    </h3>
+
+    <form class="input-group" id="search-form" role="search"></form>
+
+    <div class="d-flex justify-content-center mt-5 d-none" id="spinner">
+      <div class="spinner-border" role="status">
+        <span class="sr-only">Loading...</span>
+      </div>
+    </div>
+
+    <div
+      id="search-result"
+      class="search-result-list justify-content-center"
+    ></div>
+
+    <script src="js/script.js"></script>
+    <script src="js/marquee.js"></script>
+    <script src="js/searchForm.js"></script>
+    <script src="js/searchResults.js"></script>
+    <script>
+      (async function () {
```

Comment:
something about baseURL - you can declare it here, and then use it across all files, instead of declearing it inside every file. DRY